guides/_posts/2014-01-5-vfl.html

```diff
@@ -300,34 +301,144 @@
 
 {% highlight css %}
 /* VFL */
-@horizontal [#flexibleButton(>= 70, <= 100))];
+@h (#flexibleButton(>= 70, <= 100)));
 
 /* Equivalent CCSS */
 #flexibleButton[width] >= 70;
 #flexibleButton[width] <= 100;
 {% endhighlight %}
 
-<img class="vfl-small-img" src="../../assets/images/vfl/completeLayout.png" width="315" height="27" alt="CompleteLayout">
+
+## VFL Selectors
+All previous example of VFL expressions use basic [selectors](/guides/selectors). But you can also use combinators in VFL expression
+like this:
 
 {% highlight css %}
-/* VFL */
-@horizontal |-24-[#find]-[#findNext]-[#findField(>=30)]-24-| in(#popup);
+@v | (.featured > article ~ .title) | in(#elm > .class .class2"_fallout")
```

Comment:
In any case, I propose that we call the virtual `"virtual"` or something less obscure.